front/app/api/graph_data_units/types.ts

```diff
@@ -3,39 +3,110 @@ import { Moment } from 'moment';
 
 // live
 export type ResolvedName =
-  | 'ReactionsByTimeWidget'
   | 'SurveyResultsWidget'
-  | 'CommentsByTimeWidget'
-  | 'PostsByTimeWidget'
-  | 'ActiveUsersWidget'
+  | 'MostReactedIdeasWidget'
   | 'VisitorsWidget'
   | 'VisitorsTrafficSourcesWidget'
   | 'GenderWidget'
   | 'AgeWidget'
-  | 'MostReactedIdeasWidget';
+  | 'ActiveUsersWidget'
+  | 'PostsByTimeWidget'
+  | 'CommentsByTimeWidget'
+  | 'ReactionsByTimeWidget';
 
-export interface ParametersLive {
+export interface BaseParams {
   resolvedName: ResolvedName;
-  props: PropsLive;
+  props: object;
```

Comment:
```suggestion
  props: Record<string, any>;
```

Not a huge issue but using `object` is not recommended, because it's defined as anything that's not a primitive (see https://www.typescripttutorial.net/typescript-tutorial/typescript-object-type/).